tests/Rector/ClassMethod/RemoveDefaultGetBlockPrefixRector/Fixture/extends.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\ClassMethod\RemoveDefaultGetBlockPrefixRector\Fixture;
+
+use Symfony\Component\Form\AbstractType;
+
+class IconType extends ChoiceType
+{
+    public function getBlockPrefix()
+    {
+        return 'icon';
+    }
+}
+
+class ChoiceType extends AbstractType
+{
+    public function getBlockPrefix()
+    {
+        return 'choice';
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Symfony\Tests\Rector\ClassMethod\RemoveDefaultGetBlockPrefixRector\Fixture;
+
+use Symfony\Component\Form\AbstractType;
+
+class IconType extends ChoiceType
+{
+    public function getBlockPrefix()
+    {
+        return 'icon';
+    }
+}
+
+class ChoiceType extends AbstractType
+{
+    public function getBlockPrefix()
+    {
+        return 'choice';
+    }
+}
+
+?>
```

Comment:
you can remove second part if before and after is equal, also, please rename fixture file with `skip_` prefix, eg: `skip_with_get_block_prefix.php.inc`

Reply:
This fixture is wrong. It's just an example requested by https://github.com/rectorphp/rector/issues/7347

Reply:
What is the right one?

Reply:
It's complicated! see this [comment](https://github.com/rectorphp/rector-symfony/pull/212#discussion_r938984638)


But I have an idea!

Reply:
Pushed, the test is now failing as expected, and the fixtures is OK, no need to update it.